docs/plugin_developer_guide.md

```diff
@@ -75,6 +77,7 @@ IBM Cloud CLI SDK provides a set of APIs to register and manage plug-ins. It als
     - _Version_: The version of plug-in.
     - _MinCliVersion_: The minimal version of IBM Cloud CLI required by the plug-in.
     - _PrivateEndpointSupported_: Indicates if the plug-in is designed to also be used over the private network.
+    - _IsCobraPlugin_: Indicates if the plug-in is built using the Cobra framework
```

Comment:
**Callout**: This was missed in previous PR #332 . We should add in the guide on the new metadata field that will be added